CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- **BREAKING** bump minimum Node version to 16 ([#114](https://github.com/MetaMask/create-release-branch/pull/114))
+- Reorder workflow to update changelogs first ([#109](https://github.com/MetaMask/create-release-branch/pull/109))
+  -  when you run this tool you can use the changelogs to decide which versions to include in your release.
+- Allow users to omit packages without the need for `intentionally-skip` ([#98](https://github.com/MetaMask/create-release-branch/pull/98))
+- Compel users to release packages with breaking changes alongside their dependents ([#101](https://github.com/MetaMask/create-release-branch/pull/101))
+- Compel users to release new versions of dependencies alongside their dependents ([#102](https://github.com/MetaMask/create-release-branch/pull/102))
+
+### Fixed
+- fix: replace `null` with `intentionally-skip` in release-spec template comment ([#99](https://github.com/MetaMask/create-release-branch/pull/99))
```

Comment:
I reviewed this PR and I'm not sure this is something we actually want. I've submitted a new PR to address this: https://github.com/MetaMask/create-release-branch/pull/119. When this is merged, then this point will go away, so I think we can remove it from the changelog:

``` suggestion
```